src/Mailgun/Resource/Api/Message/ShowResponse.php

```diff
@@ -371,14 +363,6 @@ public function getContentIdMap()
     }
 
     /**
-     * @param string $contentIdMap
-     */
-    private function setContentIdMap($contentIdMap)
```

Comment:
I agree that we should not have dead code.. 
It was a bug that those functions are not used. Removing those functions does not resolve the bug. I would much rather see those functions to still be there and we fix the bug.